doc/manuals/user/transient_entities.md

```diff
@@ -0,0 +1,124 @@
+# Transient Entities
+
+Orion Context Broker implements *transient entities*. A transient entity is like any other entity 
+(i.e. it has id/type, a set of attributes, etc.) but with a limited lifetime, so when a given
+time has come, the entity is automatically deleted from the context managed by Orion.
+
+Thus, a first and very important piece of advice: **be careful if you use transient entities as once
+the expiration time has come, the entity will be automatically deleted from database and there is
+no way of recovering it**. Ensure the information you set in a transient entity is actually "transient"
+(i.e. it is not important after the lifetime of the entity has passed and it gets expired).
+
+In addition, **have a look to the [backward compatibility considerations section](#backward-compatibility-considerations)
+in the case you are already using attributes with the exact name `dateExpires`**.
+
+## The `dateExpires` attribute
+
+Transient entities are based in the `dateExpires` attribute. This is an attribute of `DateTime` type,
+according to the [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable/). 
+Its value is the datetime when the entity will expire.
+
+## Valid transitions
+
+### Create entity with `dateExpires` attribute
+
+An entity is created with transient nature if it includes the `dateExpires` attribute. For instance:
+
+```
+POST /v2/entities
+{
+  "id": "t1",
+  "type": "Ticket",
+  ...
+  "dateExpires": {
+    "value": "2028-07-07T21:35:00Z",
+    "type": "DateTime"
+  }
+}
+```
+
+creates an entity that will expire at July 7th, 2028 at 21:35.
+
+Additional considerations:
+
+* `dateExpires` has to have a valid `DateTime` value (check NGSIv2 specification for details).
+Otherwise a 400 Bad Request would be returned.
+
+* If `dateExpires` is set in the past, the entity is created expired (a bit weird, but functionally correct).
```

Comment:
how the hell it can be created expired?? shouldn't this be a 400 Error?

Reply:
It can created expired, and MongoDB can take until 60 seconds to return it. Note we are mostly delegating on MongoDB for transient entities deletion, in order to keep Orion code simpler (otherwise, we would need to add more checkings in Orion).